ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/Words.java

```diff
@@ -31,10 +32,10 @@ private Words() {}
    * @return A VM word containing {@code address} (left-padded as according to the VM specification
    *     (Appendix H. of the Yellow paper)).
    */
-  public static Bytes32 fromAddress(final Address address) {
+  public static UInt256 fromAddress(final Address address) {
```

Comment:
seems like toAddress should take UInt256 for consistency sake.  non-blocking

Reply:
UInt256 now implements Bytes32, so it's ok that way around.